frontend/src/layouts/page/page.tsx

```
@@ -3,34 +1,107 @@
+          : (
+            <div
+              className={classNames({
+                'page-container': stickyHeader,
+              })}
+            >
+              {children}
+            </div>
+          )
+      )}
+
+      <div id="floating-container"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)